### PR TITLE
Admin can add new group

### PR DESCRIPTION
**Pull request by volodymyro-in6k**
Added Group controller for mapping to admin group manage page.
Moved  rest methods into separated Admin Rest controller class.
Added angular js libs
Added angular configuration file
Configured angular admin module
Added and Configured angular goups controller
Added and Configured angular derictive
Added   service for 'group' model with creation group request
Extend nav.jsp with link for groups page
Extended dashboard layout

### CODE REVIEW

File: src/main/java/com/university/controller/AdminApiController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+public class AdminApiController {
+
+    @Autowired
+    private GroupService groupService;
+
+    @RequestMapping(value = "/groups", method = RequestMethod.POST)
+    @ResponseBody
+    public ResponseEntity<TextResponse> createGroup(@RequestBody SimpleGroupBean group) {
+
+        try{
```
Comment (alexanderko) [inline src/main/java/com/university/controller/AdminApiController.java]: incorrect formatting

File: src/main/java/com/university/controller/AdminApiController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+            groupService.create(group.getName());
+            return new ResponseEntity<TextResponse>(new TextResponse("Group successful created!"), HttpStatus.OK);
+
+        }catch (GroupNameUniqueException ex) {
```
Comment (alexanderko) [inline src/main/java/com/university/controller/AdminApiController.java]: incorrect formatting

File: src/main/webapp/WEB-INF/jsp/admin/group/groups.jsp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                    </div>
+                    <button type="submit" class="btn btn-success btn-block" ng-disabled="groupForm.$invalid">Add</button>
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
+    $('#addGroupSpan').webuiPopover({title:'Add group',url:'#addGroupForm',placement:'right',width:350});
```
Comment (alexanderko) [inline src/main/webapp/WEB-INF/jsp/admin/group/groups.jsp]: it not ok to mix angular and jquery in one place

File: src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -24,12 +24,12 @@
...
         <error-code>404</error-code>
         <location>/WEB-INF/jsp/error/404.jsp</location>
     </error-page>
     <error-page>
         <error-code>403</error-code>
         <location>/WEB-INF/jsp/error/403.jsp</location>
-    </error-page>
-</web-app>
\ No newline at end of file
+    </error-page>-->
```
Comment (alexanderko) [inline src/main/webapp/WEB-INF/web.xml]: remove commented code

File: src/main/webapp/WEB-INF/jsp/admin/group/groups.jsp (hunk 3 above)
Comment (volodymyro-in6k) [inline src/main/webapp/WEB-INF/jsp/admin/group/groups.jsp]: Thx, i have find some solution and will try the luck https://angular-ui.github.io/bootstrap/

File: src/main/webapp/resources/css/panel.css [hunk 5]
```diff
@@ -49,4 +49,4 @@ body{
 
 .panel-container{
   margin-top: 50px !important;
-}
+}
```
Comment (alexanderko) [inline src/main/webapp/resources/css/panel.css]: files should end with blank line

File: src/main/webapp/resources/js/admin/groups/groups.controller.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('university.admin')
+        .controller('GroupsController', GroupsController);
+
+    GroupsController.$inject = ['$scope', 'GroupsUtils','Flash'];
```
Comment (alexanderko) [inline src/main/webapp/resources/js/admin/groups/groups.controller.js]: missed space after `,`

File: src/main/webapp/resources/js/admin/groups/groups.controller.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+    function GroupsController($scope, $GroupsUtils, Flash) {
```
Comment (alexanderko) [inline src/main/webapp/resources/js/admin/groups/groups.controller.js]: `$` is used for angular services only

File: src/main/webapp/resources/js/admin/groups/groups.service.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
+(function () {
+    'use strict';
+
+    angular.module('university.admin')
+        .factory('GroupsUtils', GroupsUtils);
+
+    GroupsUtils.$inject = [
+        '$q', '$http'
+    ];
```
Comment (alexanderko) [inline src/main/webapp/resources/js/admin/groups/groups.service.js]: use consistent code style

File: src/main/webapp/resources/js/admin/groups/groups.service.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+    function GroupsUtils($q, $http) {
+        var service = {
+            create:create,
```
Comment (alexanderko) [inline src/main/webapp/resources/js/admin/groups/groups.service.js]: missed ` `

**Comment by alexanderko** [comment]
@volodymyro-in6k reviewed

**Pull request merged**